### DISCUSSION TITLE
OCP file of negative electrode

### DISCUSSION THREAD

**Discussion by ssingh-ipa** [pybamm-team/PyBaMM#1487] [Q&A] [answered]
Regarding the OCP csv provided in https://github.com/pybamm-team/PyBaMM/blob/main/pybamm/input/parameters/lithium-ion/negative_electrodes/graphite_Chen2020/graphite_LGM50_ocp_Chen2020.csv 
Is this the OCP vs SOC table or OCP vs discharge capacity? What is the the other column in this table?

**Comment by valentinsulzer** [comment, marked as answer]
Negative electrode stoichiometry (also called negative electrode SOC). It's concentration divided by maximum concentration
